.env.example

```
@@ -1,5 +1,11 @@
 REACT_APP_IMAGE_PATH="s3://kpmp-knowledge-environment/"
 REACT_APP_API_HOST="https://qa-atlas.kpmp.org"
+# Elasticsearch base URL, for example: https://search.example.org:9200
+REACT_APP_SEARCH_ENDPOINT=
+# Elasticsearch API key with read access to the spatial-viewer index
+REACT_APP_SEARCH_KEY=
+REACT_APP_RUN_ANALYZER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_RUN_ANALYZER key should go before the REACT_APP_SEARCH_ENDPOINT key

(UnorderedKey)